runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersListView/UsersListEdit/UsersListEdit.styles.ts

```diff
@@ -0,0 +1,26 @@
+import { LoadingButton } from '@mui/lab';
+import { Button, DialogActions } from '@mui/material';
+import styled from 'styled-components';
+
+export const StyledButton = styled(Button)`
+    width: 80px;
+    height: 40px;
```

Comment:
are you sure that we need to specify button height?

Reply:
You're right, it's not necessary, I will delete that line.